### ISSUE TITLE
SOLVED: Image upload

### ISSUE THREAD

**Issue by gazzerman** [ainterpreting/mosaico-php-backend#23]
OK after wanting to throw my machine out the window I finally solved the issue everyone seems to have on here.

Its a messed up way to solve the issue but it works and maybe someone else can write some new code to make it neat again.

I uploaded the editor.html to the root directory, which overwrites the old one (make sure the fileuploadConfig is set to the correct directory)

Then I created /backend-php/ and uploaded just these 3 files, config.php, index.php and premailer.php

Then I created three directories in the root (permissions 0777)
/dl/
/img/
/upload/

Then I created a .htaccess file

<IfModule mod_rewrite.c>
RewriteEngine On
RewriteRule (.*) /mosaico-master/backend-php/index.php [QSA,L]
</IfModule>

And here is the bad coded bit, I uploaded  .htaccess in each of those three directories I created (dl,img/upload).

Then it all worked.

So basically something is going wrong with the .htaccess file

Hope that saves someone else here the 6 hrs it took me to figure it out, lol

**Comment by perlpty** [comment]
excellent solution!!! It worked for me.
